src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -0,0 +1,70 @@
+package org.interledger.core;
```

Comment:
Oh interesting, thanks for clarifying - that makes sense.

I'm open to reverting the package name, but my only remaining concern is that "ILP" and "Interledger Protocol" are semantically equivalent, but we're trying make them mean two different things.

More specifically, the term "**ILP**" is actually an acronym that expands to "Interledger Protocol" (I think).  So, on the one hand we're saying a related protocol like "[SPSP](https://github.com/interledger/rfcs/tree/master/0009-simple-payment-setup-protocol)" (RFC-9) is **not** "[ILP](https://github.com/interledger/rfcs/tree/master/0003-interledger-protocol)" (RFC-3), yet we're also saying that SPSP **is** an "Interledger Protocol" or "ILP."

Seems like conceiving of the thing defined in [RFC-3](https://github.com/interledger/rfcs/tree/master/0003-interledger-protocol) as something like "Interledger Core" or "Interledger Core Protocol" would then allow us to say that SPSP, ILQP, etc are all `ILP`/`Interledger`.

Thoughts?

(I'm hesitant to be bike-shedding here, so feel free to respond back with 'hey, revert the package name and log a ticket to discuss later, or never'.  I'm just trying form a mental model of how to organize our code.)